### PR TITLE
B Tree

### PR DESCRIPTION
**Pull request by akhileshsantoshwar**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Created new program
- 

#### Languages Used:
- Python
- 

#### Files Added:
- B tree in python
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please complete the PR template

**Comment by akhileshsantoshwar** [comment]
> please complete the PR template

Done

**Comment by asha15** [comment]
you didn't specify the issue number next `Fixes#`, please correct it

File: B and B+ Tree in Python/B Tree [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        while queue != []:
+            popped = queue.pop(0)
+            for child in popped.children:
+                queue.append(child)
+            print(popped.key, end=' ')
+ 
+def main(): 
+    tree = None
+    
+    print('Templates for commands are as follows:(this assumes no duplicate keys)')
```
Comment (SakshiShreya) [inline B and B+ Tree in Python/B Tree]: Add a space after : (colon)

File: B and B+ Tree in Python/B Tree [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                print('BFS traversal display: ', end='')
+                tree.bfs_display()
+                print()
+            else:
+                print('Tree is empty.')
+    
+        elif operation == 'quit':
+            break
+
+
```
Comment (SakshiShreya) [inline B and B+ Tree in Python/B Tree]: Remove the extra new_line.

File: B and B+ Tree in Python/B Tree [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+main()
```
Comment (SakshiShreya) [inline B and B+ Tree in Python/B Tree]: Add a sample input and output at the end of the file.

**Comment by MastersAbh** [comment]
Hi @akhileshsantoshwar I dont see any update from your side for over 28 days, hence closing this PR.

**Pull request closed without merging**